app/controllers/whitehall_media_controller.rb

```diff
@@ -1,56 +1,31 @@
-class WhitehallMediaController < BaseMediaController
-  def download
-    if redirect_to_draft_assets_host_for?(asset)
-      redirect_to_draft_assets_host
-      return
-    end
+class WhitehallMediaController < MediaController
 
-    unless asset.accessible_by?(current_user)
-      head :forbidden
-      return
-    end
+protected
 
-    if asset.infected?
-      error_404
-      return
-    end
+  def asset
+    @asset ||= WhitehallAsset.from_params(
+      path: params[:path], format: params[:format], path_prefix: 'government/uploads/'
+    )
+  end
 
-    if asset.redirect_url.present?
-      redirect_to asset.redirect_url
-      return
-    end
+  def asset_servable?
+    not asset.infected?
```

Comment:
Although it doesn't matter in this case, I think it's better to use `!` because that has more expected precedence rules than `not` which has the lowest.

https://stackoverflow.com/questions/27429711/can-someone-explain-not-vs-in-ruby